CustomerAPI/Validators/CustomerValidator.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Data.Entities;
+using FluentValidation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Data.Validators
+{
+    public class CustomerValidator : AbstractValidator<Customer>
+    {
+        public CustomerValidator()
+        {
+
+            RuleFor(customer => customer.FullName)
+                .NotEmpty().WithMessage("Name cannot remain empty")
+                .MinimumLength(4);
+
+            RuleFor(customer => customer.Email)
+                .NotEmpty().WithMessage("Email cannot remain empty")
+                .EmailAddress().WithMessage("Email is not valId");
```

Comment:
Validações próprias do FluentValidation não precisa sobrescrever suas mensagem pois são bem descritivas
```suggestion
                .NotEmpty()
                .EmailAddress().WithMessage("Email is not valId");
```